### PR TITLE
New: Add Missing Translations

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Add various missing translate{} and associated keys as applicable

#### Todos
- [x] Translation Keys

### Reviewed the Following
- [x] Activity
- [x] AddMovie
- [x] App
- [x] Calendar
- [x] Commands
- [ ] Components (**Assuming None**)
- [x] Content
- [x] Discover Movie
- [ ] Helpers (**Assuming None**)
- [x] Interactive Import
- [x] Interactive Search
- [x] Movie
- [x] Movie File
- [x] Organize
- [x] Root Folder
- [x] Settings
- [x] Shared (**Confirmed None**)
- [ ] Store (**Assuming None**)
- [ ] Styles (**Assuming None**)
- [x] System
- [ ] Utilities (**Assuming None**)
- [x] index.js
- [x] polyfills.js
- [x] preload.js
- [x] jsconfig.json

To Do:
- [ ] frontend\src\Movie\Editor\Delete\DeleteMovieModalContent.js (conditionals)

### CODE REVIEW

File: frontend/src/Activity/Queue/RemoveQueueItemsModal.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class RemoveQueueItemsModal extends Component {
 
           <ModalBody>
             <div className={styles.message}>
-              Are you sure you want to remove {selectedCount} item{selectedCount > 1 ? 's' : ''} from the queue?
+              {translate('AreYouSureYouWantToRemoveSelectedItemsFromQueue', [selectedCount], [selectedCount > 1 ? 's' : ''])}
```
Comment (austinwbest) [inline frontend/src/Activity/Queue/RemoveQueueItemsModal.js:90]: this should not have multiple brackets.. [var, var, var]

File: frontend/src/DiscoverMovie/Exclusion/ExcludeMovieModalContent.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class ExcludeMovieModalContent extends Component {
 
         <ModalBody>
           <div className={styles.pathContainer}>
-            Exclude {title}? This will prevent Radarr from adding automatically via list sync.
+            {translate('ExcludeTitle', { title })}
```
Comment (austinwbest) [inline frontend/src/DiscoverMovie/Exclusion/ExcludeMovieModalContent.js:41]: square bracket needed

**Review by austinwbest — COMMENTED** [review]
Didn't mean to hit review but owell

File: frontend/src/Settings/Tags/Details/TagDetailsModalContent.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ function TagDetailsModalContent(props) {
   return (
     <ModalContent onModalClose={onModalClose}>
       <ModalHeader>
-        Tag Details - {label}
+        {translate('TagDetails', { label })}
```
Comment (austinwbest) [inline frontend/src/Settings/Tags/Details/TagDetailsModalContent.js:32]: square brackets

File: frontend/src/Settings/Tags/Details/TagDetailsDelayProfile.js [hunk 4]
```diff
@@ -20,16 +21,16 @@ function TagDetailsDelayProfile(props) {
...
+            translate('UsenetDisabled')
         }
       </div>
 
       <div>
         {
           enableTorrent ?
-            `Torrent Delay: ${torrentDelay}` :
-            'Torrents disabled'
+            translate('TorrentDelayTime', { torrentDelay }) :
```
Comment (austinwbest) [inline frontend/src/Settings/Tags/Details/TagDetailsDelayProfile.js:32]: square brackets

File: frontend/src/Settings/Tags/Details/TagDetailsDelayProfile.js [hunk 5]
```diff
@@ -20,16 +21,16 @@ function TagDetailsDelayProfile(props) {
       <div>
         {
           enableUsenet ?
-            `Usenet Delay: ${usenetDelay}` :
-            'Usenet disabled'
+            translate('UsenetDelayTime', { usenetDelay }) :
```
Comment (austinwbest) [inline frontend/src/Settings/Tags/Details/TagDetailsDelayProfile.js:24]: square brackets

File: frontend/src/System/Status/DiskSpace/DiskSpace.js [hunk 6]
```diff
@@ -30,6 +30,7 @@ const columns = [
   },
   {
     name: 'progress',
+    label: translate('Progress'),
```
Comment (austinwbest) [inline frontend/src/System/Status/DiskSpace/DiskSpace.js:33]: remove

File: frontend/src/System/Tasks/Queued/QueuedTasks.js [hunk 7]
```diff
@@ -10,36 +10,37 @@ import QueuedTaskRowConnector from './QueuedTaskRowConnector';
...
   },
   {
     name: 'duration',
-    label: 'Duration',
+    label: translate('Duration'),
     isVisible: true
   },
   {
     name: 'actions',
+    label: translate('Actions'),
```
Comment (austinwbest) [inline frontend/src/System/Tasks/Queued/QueuedTasks.js:43]: remove

File: frontend/src/System/Tasks/Scheduled/ScheduledTasks.js [hunk 8]
```diff
@@ -10,31 +10,32 @@ import ScheduledTaskRowConnector from './ScheduledTaskRowConnector';
...
   },
   {
     name: 'nextExecution',
-    label: 'Next Execution',
+    label: translate('NextExecution'),
     isVisible: true
   },
   {
     name: 'actions',
+    label: translate('Actions'),
```
Comment (austinwbest) [inline frontend/src/System/Tasks/Scheduled/ScheduledTasks.js:38]: remove

File: frontend/src/Settings/Metadata/Options/MetadataOptions.js [hunk 9]
```diff
@@ -9,6 +9,8 @@ import LoadingIndicator from 'Components/Loading/LoadingIndicator';
 import { inputTypes } from 'Helpers/Props';
 import translate from 'Utilities/String/translate';
 
+// Note: Do Not Translate Certifcation Countries
```
Comment (austinwbest) [inline frontend/src/Settings/Metadata/Options/MetadataOptions.js:12]: typo :p

File: frontend/src/Settings/DownloadClients/DownloadClients/DownloadClient.js [hunk 10]
```diff
@@ -89,7 +89,7 @@ class DownloadClient extends Component {
                 kind={kinds.DISABLED}
                 outline={true}
               >
-                Priority: {priority}
+                {translate('PrioritySettings', { priority })}
```
Comment (austinwbest) [inline frontend/src/Settings/DownloadClients/DownloadClients/DownloadClient.js:92]: square brackets

File: frontend/src/Settings/Tags/Details/TagDetailsModalContent.js (hunk 3 above)
Comment (bakerboy448) [reply frontend/src/Settings/Tags/Details/TagDetailsModalContent.js:32]: ` {translate('TagDetails', [label])}`

File: frontend/src/Settings/CustomFormats/CustomFormats/Specifications/Specification.js [hunk 11]
```diff
@@ -118,7 +118,7 @@ class Specification extends Component {
           message={
             <div>
               <div>
-                Are you sure you want to delete format tag '{name}'?
+                {translate('AreYouSureYouWantToDeleteFormat', { name })}
```
Comment (austinwbest) [inline frontend/src/Settings/CustomFormats/CustomFormats/Specifications/Specification.js:121]: square brackets

File: frontend/src/Settings/CustomFormats/CustomFormats/CustomFormat.js [hunk 12]
```diff
@@ -154,7 +154,7 @@ class CustomFormat extends Component {
           message={
             <div>
               <div>
-                Are you sure you want to delete custom format '{name}'?
+                {translate('AreYouSureYouWantToDeleteFormat', { name })}
```
Comment (austinwbest) [inline frontend/src/Settings/CustomFormats/CustomFormats/CustomFormat.js:157]: square brackets

File: frontend/src/RootFolder/RootFolders.js [hunk 13]
```diff
@@ -24,6 +24,7 @@ const rootFolderColumns = [
   },
   {
     name: 'actions',
+    label: translate('Actions'),
```
Comment (austinwbest) [inline frontend/src/RootFolder/RootFolders.js:27]: remove

File: frontend/src/Movie/MoveMovie/MoveMovieModal.js [hunk 14]
```diff
@@ -40,19 +41,19 @@ function MoveMovieModal(props) {
...
+          {translate('MoveFiles')}
         </ModalHeader>
 
         <ModalBody>
           {
             destinationRootFolder ?
-              `Would you like to move the movie folders to '${destinationRootFolder}'?` :
-              `Would you like to move the movie files from '${originalPath}' to '${destinationPath}'?`
+              translate('MovieFolders1', destinationRootFolder) :
+              translate('MovieFolders2', { originalPath }, { destinationPath })
```
Comment (austinwbest) [inline frontend/src/Movie/MoveMovie/MoveMovieModal.js:51]: square brackets & both vars inside a single set of brackets

File: frontend/src/Movie/MoveMovie/MoveMovieModal.js [hunk 15]
```diff
@@ -40,19 +41,19 @@ function MoveMovieModal(props) {
...
-          Move Files
+          {translate('MoveFiles')}
         </ModalHeader>
 
         <ModalBody>
           {
             destinationRootFolder ?
-              `Would you like to move the movie folders to '${destinationRootFolder}'?` :
-              `Would you like to move the movie files from '${originalPath}' to '${destinationPath}'?`
+              translate('MovieFolders1', destinationRootFolder) :
```
Comment (austinwbest) [inline frontend/src/Movie/MoveMovie/MoveMovieModal.js:50]: missing brackets

File: frontend/src/Movie/Editor/Organize/OrganizeMovieModalContent.js [hunk 16]
```diff
@@ -21,20 +21,20 @@ function OrganizeMovieModalContent(props) {
...
+          {translate('PreviewRenameHelpText')}
           <Icon
             className={styles.renameIcon}
             name={icons.ORGANIZE}
           />
         </Alert>
 
         <div className={styles.message}>
-          Are you sure you want to organize all files in the {movieTitles.length} selected movie(s)?
+          {translate('OrganizeConfirm', movieTitles.length)}
```
Comment (austinwbest) [inline frontend/src/Movie/Editor/Organize/OrganizeMovieModalContent.js:37]: missing brackets

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 17]
```diff
@@ -108,23 +108,23 @@ class DeleteMovieModalContent extends Component {
           {
             deleteFiles &&
               <div className={styles.deleteFilesMessage}>
-                <div>The movie folder <strong>{path}</strong> and all it's content will be deleted.</div>
+                <div>{translate('DeleteTheMovieFolder', { path })}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:111]: square brackets

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 18]
```diff
@@ -66,20 +66,20 @@ class DeleteMovieModalContent extends Component {
...
+      deleteFilesHelpText = translate('DeleteMovieFolderHelpText');
     }
 
     return (
       <ModalContent
         onModalClose={onModalClose}
       >
         <ModalHeader>
-          Delete - {title}
+          {translate('DeleteHeader', { title })}
```
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:82]: square brackets

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 19]
```diff
@@ -66,20 +66,20 @@ class DeleteMovieModalContent extends Component {
     const deleteFiles = this.state.deleteFiles;
     const addImportExclusion = this.state.addImportExclusion;
 
-    let deleteFilesLabel = `Delete ${movieFileCount} Movie Files`;
-    let deleteFilesHelpText = 'Delete the movie files and movie folder';
+    let deleteFilesLabel = translate('DeleteFilesLabel', { movieFileCount });
```
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:69]: square brackets

File: frontend/src/InteractiveImport/Interactive/InteractiveImportModalContent.js [hunk 20]
```diff
@@ -198,25 +198,26 @@ class InteractiveImportModalContent extends Component {
...
     const selectedIds = this.getSelectedIds();
-    const errorMessage = getErrorMessage(error, 'Unable to load manual import items');
+    const errorMessage = getErrorMessage(error, translate('UnableToLoadManualImportItems'));
 
     const bulkSelectOptions = [
-      { key: SELECT, value: 'Select...', disabled: true },
-      { key: LANGUAGE, value: 'Select Language' },
-      { key: QUALITY, value: 'Select Quality' }
+      {
+        key: SELECT, value: translate('SelectDotDot'), disabled: true },
```
Comment (austinwbest) [inline frontend/src/InteractiveImport/Interactive/InteractiveImportModalContent.js:205]: this shouldn't be dropped a line

File: frontend/src/InteractiveImport/Folder/InteractiveImportSelectFolderModalContent.js [hunk 21]
```diff
@@ -17,15 +17,15 @@ import styles from './InteractiveImportSelectFolderModalContent.css';
...
   },
   {
     name: 'lastUsed',
-    label: 'Last Used'
+    label: translate('LastUsed')
   },
   {
     name: 'actions',
-    label: ''
+    label: translate('Actions')
```
Comment (austinwbest) [inline frontend/src/InteractiveImport/Folder/InteractiveImportSelectFolderModalContent.js:28]: put back to ''

File: frontend/src/DiscoverMovie/Menus/DiscoverMovieSortMenu.js [hunk 22]
```diff
@@ -52,7 +53,7 @@ function DiscoverMovieSortMenu(props) {
           sortDirection={sortDirection}
           onPress={onSortSelect}
         >
-          In Cinemas
+          {translate('InCinemas') }
```
Comment (austinwbest) [inline frontend/src/DiscoverMovie/Menus/DiscoverMovieSortMenu.js:56]: extra space

File: frontend/src/App/AppUpdatedModalContent.js [hunk 23]
```diff
@@ -26,27 +26,25 @@ function AppUpdatedModalContent(props) {
...
-        </div>
+        <div dangerouslySetInnerHTML={{ __html: translate('VersionUpdateText', [`<span className=${styles.version}>${version}</span>`]) }} />
 
         {
           isPopulated && !error && !!update &&
             <div>
               {
                 !update.changes &&
-                  <div className={styles.maintenance}>Maintenance release</div>
+                  <div className={styles.maintenance}>{translate('MaintenanceRelease')}</div>
```
Comment (austinwbest) [inline frontend/src/App/AppUpdatedModalContent.js:40]: drop lines

File: frontend/src/AddMovie/ImportMovie/SelectFolder/ImportMovieSelectFolder.js [hunk 24]
```diff
@@ -32,6 +32,7 @@ const rootFolderColumns = [
   },
   {
     name: 'actions',
+    label: translate('Actions'),
```
Comment (austinwbest) [inline frontend/src/AddMovie/ImportMovie/SelectFolder/ImportMovieSelectFolder.js:35]: remove

File: frontend/src/AddMovie/ImportMovie/Import/ImportMovie.js [hunk 25]
```diff
@@ -116,7 +116,7 @@ class ImportMovie extends Component {
             rootFoldersPopulated &&
             !unmappedFolders.length ?
               <div>
-                All movies in {path} have been imported
+                {translate('AllMoviesInPathHaveBeenImported', path)}
```
Comment (austinwbest) [inline frontend/src/AddMovie/ImportMovie/Import/ImportMovie.js:119]: missing brackets

File: frontend/src/InteractiveImport/Language/SelectLanguageModalContent.js [hunk 26]
```diff
@@ -128,7 +128,7 @@ class SelectLanguageModalContent extends Component {
             kind={kinds.SUCCESS}
             onPress={this.onLanguageSelect}
           >
-            Select Languges
+            {translate('SelectLanguges')}
```
Comment (austinwbest) [inline frontend/src/InteractiveImport/Language/SelectLanguageModalContent.js:131]: typo

File: frontend/src/InteractiveSearch/InteractiveSearchRow.js [hunk 27]
```diff
@@ -35,12 +35,12 @@ function getDownloadTooltip(isGrabbing, isGrabbed, grabError) {
...
     return '';
   } else if (isGrabbed) {
-    return 'Added to downloaded queue';
+    return translate('AddedToDownloadQueue');
   } else if (grabError) {
     return grabError;
   }
 
-  return 'Add to downloaded queue';
+  return translate('AddedToDownloadQueue');
```
Comment (austinwbest) [inline frontend/src/InteractiveSearch/InteractiveSearchRow.js:43]: this should not be the same as the above, one is "Added" and one is "Add"

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js (hunk 17 above)
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:111]: This was marked as resolved but it still isn't square brackets

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 28]
```diff
@@ -108,23 +108,23 @@ class DeleteMovieModalContent extends Component {
           {
             deleteFiles &&
               <div className={styles.deleteFilesMessage}>
-                <div>The movie folder <strong>{path}</strong> and all it's content will be deleted.</div>
+                <div>{translate('DeleteTheMovieFolder', { path })}</div>
 
                 {
                   !!movieFileCount &&
-                    <div>{movieFileCount} movie files totaling {formatBytes(sizeOnDisk)}</div>
+                    <div>{movieFileCount} {translate('MovieFilesTotaling')} {formatBytes(sizeOnDisk)}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:115]: drop lines

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js [hunk 29]
```diff
@@ -108,23 +108,23 @@ class DeleteMovieModalContent extends Component {
...
                 }
               </div>
           }
 
           <FormGroup>
-            <FormLabel>Add List Exclusion</FormLabel>
+            <FormLabel>{translate('AddListExclusion')}</FormLabel>
```
Comment (austinwbest) [inline frontend/src/Movie/Delete/DeleteMovieModalContent.js:121]: drop lines

File: frontend/src/Movie/Details/MovieDetailsLinks.js [hunk 30]
```diff
@@ -23,7 +24,7 @@ function MovieDetailsLinks(props) {
           kind={kinds.INFO}
           size={sizes.LARGE}
         >
-          TMDb
+          {translate('TMDb')}
```
Comment (austinwbest) [inline frontend/src/Movie/Details/MovieDetailsLinks.js:27]: doesn't _need_ translated

File: frontend/src/Movie/Details/MovieDetailsLinks.js [hunk 31]
```diff
@@ -36,7 +37,7 @@ function MovieDetailsLinks(props) {
           kind={kinds.INFO}
           size={sizes.LARGE}
         >
-          Trakt
+          {translate('Trakt')}
```
Comment (austinwbest) [inline frontend/src/Movie/Details/MovieDetailsLinks.js:40]: doesn't _need_ translated

File: frontend/src/Movie/Details/MovieDetailsLinks.js [hunk 32]
```diff
@@ -51,7 +52,7 @@ function MovieDetailsLinks(props) {
               kind={kinds.INFO}
               size={sizes.LARGE}
             >
-              IMDb
+              {translate('IMDb')}
```
Comment (austinwbest) [inline frontend/src/Movie/Details/MovieDetailsLinks.js:55]: doesn't _need_ translated

File: frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js [hunk 33]
```diff
@@ -48,12 +48,12 @@ class MovieTitlesTableContent extends Component {
 
         {
           !isFetching && !!error &&
-            <div className={styles.blankpad}>Unable to load alternative titles.</div>
+            <div className={styles.blankpad}>{translate('UnableToLoadAltTitle')}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js:51]: drop lines

File: frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js [hunk 34]
```diff
@@ -48,12 +48,12 @@ class MovieTitlesTableContent extends Component {
...
         }
 
         {
           isPopulated && !hasItems && !error &&
-            <div className={styles.blankpad}>No alternative titles.</div>
+            <div className={styles.blankpad}>{translate('NoAltTitle')}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js:56]: drop lines

File: frontend/src/InteractiveSearch/InteractiveSearchRow.js (hunk 27 above)
Comment (bakerboy448) [reply frontend/src/InteractiveSearch/InteractiveSearchRow.js:43]: fixed

File: frontend/src/Movie/History/MovieHistoryTableContent.js [hunk 35]
```diff
@@ -72,12 +72,12 @@ class MovieHistoryTableContent extends Component {
 
         {
           !isFetching && !!error &&
-            <div className={styles.blankpad}>Unable to load history</div>
+            <div className={styles.blankpad}>{translate('UnableToLoadHistory')}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/History/MovieHistoryTableContent.js:75]: drop lines

File: frontend/src/Movie/History/MovieHistoryTableContent.js [hunk 36]
```diff
@@ -72,12 +72,12 @@ class MovieHistoryTableContent extends Component {
...
         }
 
         {
           isPopulated && !hasItems && !error &&
-            <div className={styles.blankpad}>No history</div>
+            <div className={styles.blankpad}>{translate('NoHistory')}</div>
```
Comment (austinwbest) [inline frontend/src/Movie/History/MovieHistoryTableContent.js:80]: drop lines

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js (hunk 17 above)
Comment (bakerboy448) [reply frontend/src/Movie/Delete/DeleteMovieModalContent.js:111]: now its good

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js (hunk 28 above)
Comment (bakerboy448) [reply frontend/src/Movie/Delete/DeleteMovieModalContent.js:115]: dome

File: frontend/src/Movie/MoveMovie/MoveMovieModal.js [hunk 37]
```diff
@@ -63,14 +64,14 @@ function MoveMovieModal(props) {
...
-            No, I'll Move the Files Myself
+            {translate('NoMoveFilesSelf')}
           </Button>
 
           <Button
             kind={kinds.DANGER}
             onPress={onMoveMoviePress}
           >
-            Yes, Move the Files
+            {translate('YesMovieFiles')}
```
Comment (austinwbest) [inline frontend/src/Movie/MoveMovie/MoveMovieModal.js:74]: move files, not movie files :p

File: frontend/src/Movie/Delete/DeleteMovieModalContent.js (hunk 29 above)
Comment (bakerboy448) [reply frontend/src/Movie/Delete/DeleteMovieModalContent.js:121]: done

File: frontend/src/Movie/Details/MovieDetailsLinks.js (hunk 30 above)
Comment (bakerboy448) [reply frontend/src/Movie/Details/MovieDetailsLinks.js:27]: _shouldn't_ _need_ to be

seems future proof?

File: frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js (hunk 33 above)
Comment (bakerboy448) [reply frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js:51]: done

File: frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js (hunk 34 above)
Comment (bakerboy448) [reply frontend/src/Movie/Details/Titles/MovieTitlesTableContent.js:56]: ditto

File: frontend/src/Settings/Indexers/Indexers/Indexer.js [hunk 38]
```diff
@@ -98,7 +98,7 @@ class Indexer extends Component {
           {
             supportsRss && enableRss &&
               <Label kind={kinds.SUCCESS}>
-                RSS
+                {translate('RSS')}
```
Comment (austinwbest) [inline frontend/src/Settings/Indexers/Indexers/Indexer.js:101]: doesn't _need_ translated

File: frontend/src/Movie/History/MovieHistoryTableContent.js (hunk 35 above)
Comment (bakerboy448) [reply frontend/src/Movie/History/MovieHistoryTableContent.js:75]: lint really needs to be updated to get these -_-

File: frontend/src/Movie/History/MovieHistoryTableContent.js (hunk 36 above)
Comment (bakerboy448) [reply frontend/src/Movie/History/MovieHistoryTableContent.js:80]: +1

File: frontend/src/Movie/MoveMovie/MoveMovieModal.js (hunk 37 above)
Comment (bakerboy448) [reply frontend/src/Movie/MoveMovie/MoveMovieModal.js:74]: movie the files 👍

File: frontend/src/System/Logs/Files/LogFiles.js [hunk 39]
```diff
@@ -29,6 +29,7 @@ const columns = [
   },
   {
     name: 'download',
+    label: translate('Download'),
```
Comment (austinwbest) [inline frontend/src/System/Logs/Files/LogFiles.js:32]: remove

**Review by austinwbest — CHANGES_REQUESTED** [review]
Remember anything you change in the frontend that effects the language key needs to be changed in the en.json & remember **ammend commit** :)

**Review by austinwbest — APPROVED** [review]

**Pull request merged**